src/main/java/com/truelayer/java/commonapi/entities/UserResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.truelayer.java.commonapi.entities;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+@EqualsAndHashCode
+public class UserResponse {
```

Comment:
Should we call this `UserDetail` since it's used in `PaymentDetail` and `MandateDetail`?
Also `UserResponse` sounds too generic for me, but let me know what you guys think.

Reply:
Was wondering about that as well, don't have a strong preference. 

In c# I will probably go for `UserResponse` or `UserContract`. Here I can see we have `CreatePaymentResponse`, `PaymentDetails`. 

In Java is a 'detail' suffix usually used to identify contracts?

Reply:
> In Java is a 'detail' suffix usually used to identify contracts?

No, AFAIK there aren't guidelines on how to name such classes.
My suggestion was more to standardise the name to the classes that we already expose to clients - `PaymentDetail` and `MandateDetail`.